lib/versioned/^4.0.0/index.js

```diff
@@ -48,8 +48,8 @@ function execute(opts, env) {
       tree = gulpInst.tree({ deep: true });
       tree.label = 'Tasks for ' + chalk.magenta(tildify(env.configPath));
 
-      return logTasks(tree, function(task) {
-        return gulpInst.task(task).description;
+      return logTasks(tree, opts.depth, function(taskname) {
+        return gulpInst.task(String(taskname.toString()));
```

Comment:
@phated If the `taskname` is a function, gulp (undertaker) regists a new task. There is no such case normally, but I wanted make it sure to avoid the case. So, as you say, there is basically no problem when they removed.